rttr-zh_CN.po

```diff
@@ -33,15 +33,15 @@ msgid ""
 "Starting the game\n"
 msgstr ""
 "\n"
-"Starting the game\n"
+"开始游戏\n"
 
 #: libs/s25main/resources/ArchiveLoader.cpp:86
 msgid "  Loading %1% entries: "
-msgstr "  Loading %1% entries: "
+msgstr " 加载 %1% 条项目: "
 
 #: libs/s25main/JoinPlayerInfo.cpp:51
 msgid " (AI)"
-msgstr " (AI)"
+msgstr ""
```

Comment:
like these

Reply:
that not need  translate to Chinese

Reply:
So those are supposed to be kept. Otherwise the game will "translate" those to an empty string, i.e. doesn't display it at all.